<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_routes</class>
 <widget class="QWidget" name="add_routes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>640</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите ФИО водителя</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_fio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите дату поездки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите пункт отправления</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите авто</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_auto">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_otprav">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>Москва</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Санкт-Петербург</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Нижний Новгород</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Казань</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ростов-на-Дону</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Воронеж</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Липецк</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Киров</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите пункт назначения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите время отправления</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_nazn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Москва</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Санкт-Петербург</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Нижний Новгород</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Казань</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ростов-на-Дону</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Воронеж</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Липецк</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Киров</string>
    </property>
   </item>
  </widget>
  <widget class="QTimeEdit" name="timeEdit_otprav">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите время прибытия</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit_prib">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
